build/testing/cli.go

```diff
@@ -178,6 +178,7 @@ Available Commands:
   help        Help about any command
   import      Import flags/segments/rules from file
   migrate     Run pending database migrations
+  validate    Validate a list of flipt features.yaml files
```

Comment:
```suggestion
  validate    Validate flipt flag state {yaml, yml} files
```

I was wondering if the wording should be something of the following, since it won't always be `features.yaml`